md/UNICODE_FIX_ANALYSIS.md

```
@@ -0,0 +1,296 @@
+But if this text gets into HTML that Starlette tries to encode:
+
+```python
+html = f"<div>{text}</div>"
+html.encode('utf-8')  # May fail if text contains surrogates
+```
+
+This is where the error occurs.
+
+## References
+
+- Python Unicode Documentation: https://docs.python.org/3/howto/unicode.html
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)